Ad spend (#107)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/sources/google-ads.md

---
title: Google Ads (Beta)

> description: Amplitude Data's Google Ads integration lets you import your Google Ad spend, click and impression data with just a few clicks.

Reviewer: ```suggestion
description: Amplitude Data's Google Ads integration lets you import your Google Ad spend, click, and impression data in just a few steps.
```

---

> Amplitude Data's Google Ads integration lets you import your Google Ad spend, click and impression data with just a few clicks.

Reviewer: ```suggestion
Amplitude Data's Google Ads integration lets you import your Google Ad spend, click, and impression data in just a few steps.
```

--8<-- "includes/closed-beta.md"

## Setup

### Prerequisites

To set up, you need the following: 

> - `Google Ads Customer ID` of the ad account you would like imported (https://support.google.com/google-ads/answer/1704344?hl=en)

Reviewer: ```suggestion
- [Google Ads Customer ID](https://support.google.com/google-ads/answer/1704344?hl=en) of the ad account you want to connect to.
```

### Amplitude setup 

1. In Amplitude, navigate to **Data Sources**, then find **Google Ad** in the **I want to import data into Amplitude** tab.

![Google Add Source](../../assets/images/marketing-analytics/add-sources.png)

!!! note 
    This integration must be enabled on a per-project basis.

> 2. Log into Google and grant Amplitude permission in the consent form.

Reviewer: ```suggestion
2. Log in to Google and grant Amplitude permission in the consent form.
```

![Google Login Image](../../assets/images/marketing-analytics/google-login.png)

> 3. Enter the `Google Ads Customer ID` for all the ad accounts you would like to sync the data.

Reviewer 2: "for all the ad accounts" seems misleading here, since it's just one. Just "for the ad account you would like to sync data from."?

> 3. Enter the `Google Ads Customer ID` for all the ad accounts you would like to sync the data.

Reviewer: ```suggestion
3. Enter the Google Ads Customer ID for the ad account you want to import data from.
```

![Google Enter Account ID](../../assets/images/marketing-analytics/google-enter-info.png)

> 4. [Optional] You also have the option to do a one time iimport on the historical data in the time period of your choice.

Reviewer 2: "import"

> 4. [Optional] You also have the option to do a one time iimport on the historical data in the time period of your choice.

Reviewer: ```suggestion
4. [Optional] Import past data for a given period.
```
